src/modules/Jobs/jobCreate/containers/JobCreateForm/useJobCreateForm.tsx

```diff
@@ -234,7 +212,7 @@ export const useJobCreateForm = () => {
   };
 
   const addQuestion = (q: QuestionReq) => {
-    setQuestions(questions.concat(q));
+    setQuestions(questions.concat(q as Question));
```

Comment:
the type of `q` is QestionReq and do you concat to questions as `Question`?
P.S. can you please rename `q` to `newQuestion` to more clear?

Reply:
Because the same state and form is used for creating and updating questions. Creating questions needs `Question` type (without id), but updating needs (QuestionReq) with `id`. That's why we have cast them to each other in several points